packages/fxa-content-server/server/lib/routes/validate-email-domain.js

```diff
@@ -13,7 +13,8 @@ const joi = require('joi');
 const dns = require('dns');
 const VError = require('verror');
 const resolver = new dns.promises.Resolver();
-const results = ['MX', 'A', 'none'].reduce((accumulator, val) => {
+const config = require('../configuration');
```

Comment:
We should pass in the config for this route like the others.  There's no need to break that pattern and potentially mislead/confuse future devs.